docs/requester_pays.rst

```diff
@@ -1,268 +0,0 @@
-through this account.
-
-.. note::
-
-  If you are using a pre-existing Billing Account and Project, you might still need to
-  associate them with each other if they haven't been used together previously.
-
-At the top of the `billing page <https://console.cloud.google.com/billing>`__ there
-should be two tabs: "My Billing Accounts" and "My Projects". Select "My Projects" and
-then in the row for the new project you created above, click on the three dots in the
-far righthand column labeled "Actions". The menu that pops up should have an
-"Add Billing" or "Change Billing" option. Click on that, and you should be given the
-choice of which billing account to associate with the selected project.
-
-.. image:: images/03-enable-project-billing.png
-
-Give yourself permission to spend
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Finally, you need to give yourself (your Google ID) the power to spend money within
-the new project, using the billing method you just specified. See these instructions
-for `Using Google Cloud IAM <https://cloud.google.com/iam/docs/granting-changing-revoking-access#granting-console>`__
-(Identity and Access Management), and navigate to the
-`IAM Admin page <https://console.cloud.google.com/iam-admin/iam>`__.
-
-Your Google ID (email address) should be listed as a "Principal" and you need to grant
-that Principal the "Role" of **Service Usage Consumer**, which enables it to make
-billed requests on the behalf of the currently selected project (which should be the
-one you created at the beginning of this process -- the project should be visible in the
-dropdown menu near the upper-left corner of the page).
-
-Click on the little pencil icon in the right column of the row associated with your
-Google ID to "edit principal".
-
-.. image:: images/04-iam-admin.png
-
-This should give you the option to associate a role with a particular combination of
-Principal and Project.
-
-.. image:: images/05-add-a-role.png
-
-Click on the "Role" field and type in "Service Usage Consumer"
-and it should filter the very long list of available roles down to a few, one of which
-is "Service Usage Consumer". Select that role, and click on the Save button.
-
-.. image:: images/06-add-service-usage-consumer.png
-
-
-Check your GCP credit
-~~~~~~~~~~~~~~~~~~~~~
-Google provides new accounts with $300 in free credits so you can experiment a bit
-without spending any of your own money. Check out the
-`docs on the free trial period <https://cloud.google.com/free/docs/gcp-free-tier>`__
-to understand how it works.
-
-To check in on your remaining credits or overall billing status you can go to the
-Billing Account Overview and click on the "Credit details" link in the Credits box on
-the right hand side of the page:
-
-.. image:: images/07-billing-account-overview.png
-
-This will show you a table of any existing GCP credits that have been granted to your
-account:
-
-.. image:: images/08-credits-status.png
-
-Set up the Google Cloud SDK
```

Comment:
Folks on our team have found this Google Cloud SDK section useful for accessing nightly build outputs. I think we may want to migrate them into our main PUDL documentation in association with the development environment / nightly builds section.

Reply:
Sounds good. I'm going to add aws cli documentation instead of GCP requester pays. This way AWS will cover the egress fees when catalyst folks copy nightly build outputs to their local machines.

Reply:
Yeah no reason to use the requester pays unless we have to.

I guess any unsuccessful nightly builds we'll still have to pull down on our own dime.

Reply:
True, I can include that in our docs too.